thoth/storages/graph/postgres.py

```diff
@@ -5091,7 +5091,7 @@ def sync_solver_result(self, document: dict, *, force: bool = False) -> None:
                 ecosystem="python",
                 solver_name=solver_name,
                 solver_version=solver_version,
-                os_name=map_os_name(os_name),
+                os_name=os_name,
```

Comment:
I'm not sure what issue will this address. The `map_os_name` method maps `ubi`->`rhel` in solvers:

https://github.com/thoth-station/common/blob/5b96cd3c2ddf4856426f69f87421215643f3f4cc/thoth/common/helpers.py#L182-L187

As we do not have "solver-ubi-*" names ([quay](https://quay.io/organization/thoth-station)), this operation is (correctly) a noop in the syncing logic. The call removed just makes sure that if a solver named  `solver-ubi-*` is accidentally registered in the system, the database entries will hold correctly remapped entries as `solver-rhel-*` would be used.

Reply:
> I'm not sure what issue will this address. The `map_os_name` method maps `ubi`->`rhel` in solvers:
> 
> https://github.com/thoth-station/common/blob/5b96cd3c2ddf4856426f69f87421215643f3f4cc/thoth/common/helpers.py#L182-L187
> 
> As we do not have "solver-ubi-*" names ([quay](https://quay.io/organization/thoth-station)), this operation is (correctly) a noop in the syncing logic. The call removed just makes sure that if a solver named `solver-ubi-*` is accidentally registered in the system, the database entries will hold correctly remapped entries as `solver-rhel-*` would be used.

So we need the opposite mapping: https://github.com/thoth-station/storages/issues/2196#issuecomment-859491353
as we want to have `ubi` in the database and not `rhel`

Reply:
We can close this PR, as the logic is correctly mapping ubi entries to rhel.